CodeConverter/CSharpToVB/ExpressionVisitor.vb

```diff
@@ -1666,13 +1664,13 @@ Namespace CSharpToVBCodeConverter.DestVisualBasic
                     Dim ReportProgress As Boolean = ExpressionLastIndex > 500
 
                     If ReportProgress Then
-                        s_originalRequest.ProgressBar?.Maximum(ExpressionLastIndex + 1)
+                        s_originalRequest.Progress?.Report(New ProgressReport(0, node.Expressions.Count))
                     End If
                     ' Figuring out this without using Accept is complicated below is safe but not fast
                     Dim ItemIsField As Boolean = node.Expressions.Any AndAlso TypeOf node.Expressions(0).Accept(Me) Is VBS.FieldInitializerSyntax
                     For i As Integer = 0 To ExpressionLastIndex
                         If ReportProgress Then
-                            s_originalRequest.ProgressBar?.Increment(1)
+                            s_originalRequest.Progress?.Report(New ProgressReport(i + 1, node.Expressions.Count))
```

Comment:
I preserved the original behavior exactly, if I understand it properly. It appears that the progress bar always shows itself a step ahead of what is actually completed. I'm more than happy to change this if that's not what you intended.

Reply:
It's fine it only gets called for large numbers and you will never notice one way of the other.